### PR TITLE
Update readme

### LINKED ISSUE #172: Add numbering to UGDG
Add numbering for UGDG for easier navigation

### PR DESCRIPTION
**Pull request by Wincenttjoi**
Changes made:
- Update UI.png showing on userguide and readme, includes partitioning and labels
- Add latest messages to help start window
- Add numbering to UG
- Fix UG table summary 
- Fix UG feature introduction to outside the notes box
- Add one more paragraph to introduction for purpose of document

Fixes #172 #184 #185

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -5,48 +5,51 @@ title: User Guide
...
+as well as summary commands.
+
 
 
-## Quick start
+--------------------------------------------------------------------------------------------------------------------
+## 2. Quick start
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java `11` or above irnstalled in your Computer.
```
Comment (saad-haider-pp) [inline docs/UserGuide.md:23]: typo

File: docs/UserGuide.md [hunk 2]
```diff
@@ -5,48 +5,51 @@ title: User Guide
...
 
 OneShelf has 2 main features that it is able to store, namely are:
 1. Inventory items
 2. Pending deliveries
 
 From here onwards, the term `item` and `delivery` are used specifically for inventory items and pending deliveries respectively.
+*You may want to refer to Figure 1 above.*
 
+<div markdown="block" class="alert alert-info">
 **:information_source: Notes about the item command format:**<br>
```
Comment (saad-haider-pp) [inline docs/UserGuide.md:54]: For some reason, the **bold** markdown isn't applying in the preview

File: docs/UserGuide.md [hunk 3]
```diff
@@ -249,55 +253,61 @@ Format: `redo`
 * After any command that changes the state of data (such as add, clear, delete, edit), the new state becomes the latest state
 (i.e. the previous undo commands are "forgotten" and `redo` will have no effect)
 
-
-
-### Saving the data
+--------------------------------------------------------------------------------------------------------------------
+### 3.2 Default Features
```
Comment (saad-haider-pp) [inline docs/UserGuide.md:257]:
```suggestion
### 3.2 General Features
```

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 4]
```diff
@@ -17,20 +17,95 @@
...
+                        + "You may also put multiple keywords if you would like search a few items concurrently.\n"
+                        + "Example: `find-d n/AARON p/91231231` returns any delivery that has name AARON and anyone\n"
+                        + "who has the phone number 91231231.")
+                .append(DIVIDER)
+                .append("After entering the find feature, you can revert back to the entire list "
+                        + "by entering `list-i` or `list-d`.\n")
+                .append(DIVIDER)
+                .append("CONGRATULATIONS! You have completed OneShelf's tutorial! If you would "
+                        + "want to find out more about OneShelf,\n"
+                        + "feel free to check out the user guide's link stated above!");
```
Comment (saad-haider-pp) [inline src/main/java/seedu/address/commons/core/Messages.java:108]: This is awesome!

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 5]
```diff
@@ -51,15 +126,16 @@ public static final String getHelpSummary() {
...
                 .append("Clear all inventory items: clear-i\n")
                 .append("List all inventory items: list-i\n")
+                .append(DIVIDER)
                 .append("DELIVERY SPECIFIC COMMANDS\n")
                 .append(DIVIDER)
                 .append("Add delivery: add-d n/NAME p/PHONE a/ADDRESS o/ORDER\n")
                 .append("Delete a delivery: delete-d INDEX\n")
                 .append("Edit a delivery: edit-d INDEX [n/NAME] [p/PHONE] [a/ADDRESS] [o/ORDER]\n")
-                .append("Find a delivery: find-d KEYWORD [MORE_KEYWORDS]\n")
+                .append("Find a delivery: find-d n/KEYWORD [MORE_KEYWORDS]\n")
```
Comment (saad-haider-pp) [inline src/main/java/seedu/address/commons/core/Messages.java:138]: this implies that `n/` is mandatory. 
Maybe something like `find-d [n/NAME | p/PHONE | a/ADDRESS | o/ORDER]`

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 6]
```diff
@@ -51,15 +126,16 @@ public static final String getHelpSummary() {
                 .append("Delete an inventory item: delete-i INDEX\n")
                 .append("Edit an inventory item: edit-i INDEX [n/NAME] [q/QUANTITY]"
                         + " [s/SUPPLIER] [max/MAX_QUANTITY] [t/TAG]\n")
-                .append("Find an inventory item: find-i KEYWORD [MORE_KEYWORDS]\n")
+                .append("Find an inventory item: find-i n/KEYWORD [MORE_KEYWORDS]\n")
```
Comment (saad-haider-pp) [inline src/main/java/seedu/address/commons/core/Messages.java:129]: refer to the one for find-d

File: docs/UserGuide.md [hunk 7]
```diff
@@ -5,48 +5,51 @@ title: User Guide
...
+1. Ensure you have Java `11` or above irnstalled in your Computer.
 
 2. Download the latest `oneshelf.jar` from [here](https://github.com/AY2021S1-CS2103T-T12-1/tp/releases).
 
 3. Copy the file to the folder you want to use as the _home folder_ for your Inventory Book.
 
 4. Double-click the file to start the app. The similar GUI shown below (Figure 1) should appear in a few seconds. <br>
    *Note how the app contains some sample data but the installed version on your desktop might have a different data set.*<br>
-   ![Ui](images/Ui.png)
+   <br> ![Ui](images/Ui.png) <br>
```
Comment (saad-haider-pp) [inline docs/UserGuide.md:31]: very nice annotations!

**Review by saad-haider-pp — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 2 above)
Comment (Wincenttjoi) [reply docs/UserGuide.md:54]: image
seems to be fine on my side

Comment (saad-haider-pp) [reply docs/UserGuide.md:54]: hmm I didn't check it on the website, but on github itself it looked different

Comment (saad-haider-pp) [reply docs/UserGuide.md:54]: Like this

image

**Pull request merged**